### PR TITLE
Add initial doc for kotlin

### PR DESCRIPTION
**Pull request by talhacohen**

### CODE REVIEW

File: docs/quickstart.mdx [hunk 1]
```diff
@@ -282,6 +309,16 @@ svix.getEndpoint()
               .description("My main endpoint"));
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+val svix = Svix("AUTH_TOKEN")
+svix.endpoint
+        .create("app_Xzx8bQeOB1D1XEYmAJaRGoj0",
```
Comment (tasn) [inline docs/quickstart.mdx:318]: Extra space?

File: docs/management-ui.mdx [hunk 2]
```diff
@@ -55,6 +55,16 @@ DashboardAccessOut dashboard = svix.getAuthentication().dashboardAccess("app_Xzx
 System.out.println(dashboard.getUrl());
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+val svix = Svix("AUTH_TOKEN")
+val dashboard: DashboardAccessOut = svix.authentication.dashboardAccess("app_Xzx8bQeOB1D1XEYmAJaRGoj0")
```
Comment (tasn) [inline docs/management-ui.mdx:63]: No need to explicitly set the type of the var.

**Review by tasn — COMMENTED** [review]
Isn't there a special syntax to call suspended functions? Maybe not, but it looks weird. Shouldn't they be awaited? (Maybe not)

File: docs/management-ui.mdx (hunk 2 above)
Comment (talhacohen) [reply docs/management-ui.mdx:63]: Done.

File: docs/quickstart.mdx (hunk 1 above)
Comment (talhacohen) [reply docs/quickstart.mdx]: let me know if that what you meant

**Comment by talhacohen** [comment]
> Isn't there a special syntax to call suspended functions? Maybe not, but it looks weird. Shouldn't they be awaited? (Maybe not)

It depends from where you call it, I saw several examples with runBlocking, launch async

**Comment by svix-frank** [comment]
We need to add kotlin here for highlighting to work: `https://github.com/svix/svix-docs/blob/c6845dc0ded6c9283c70382244063c2f850f2833/docusaurus.config.js#L50`

File: docs/quickstart.mdx [hunk 3]
```diff
@@ -180,6 +192,21 @@ svix.getMessage().create("app_Xzx8bQeOB1D1XEYmAJaRGoj0",
...
+```kotlin
+val svix = Svix("AUTH_TOKEN")
+svix.message.create("app_Xzx8bQeOB1D1XEYmAJaRGoj0",
+        MessageIn("invoice.paid",
+                "{" +
+                        "\"id\": \"invoice_WF7WtCLFFtd8ubcTgboSFNql\"," +
+                        "\"status\":  \"paid\"," +
+                        "\"attempt\": 2" +
+                        "}",
+                "evt_Wqb1k73rXprtTm7Qdlr38G"))
```
Comment (svix-frank) [inline docs/quickstart.mdx:207]:
```suggestion
        MessageIn("invoice.paid",
                mapOf<String, Any>(
                        "id" to "invoice_WF7WtCLFFtd8ubcTgboSFNql",
                        "status" to "paid",
                        "attempt" to 2
                  ),
                "evt_Wqb1k73rXprtTm7Qdlr38G"))
```

File: docs/event-types.mdx [hunk 4]
```diff
@@ -246,6 +258,42 @@ public class App {
...
+                        exitProcess(1)
+                    }
+                }
+            }
+        } catch (e: IOException) {
+            println(e.toString())
+            exitProcess(1)
+        }
+    }
+}
```
Comment (svix-frank) [inline docs/event-types.mdx:294]:
```suggestion
package sviximport

import com.svix.kotlin.Svix
import com.svix.kotlin.models.EventTypeIn
import com.svix.kotlin.exceptions.ApiException
import java.io.File
import java.io.IOException
import kotlinx.coroutines.runBlocking
import kotlin.system.exitProcess

fun main() = runBlocking {
    val svix = Svix("AUTH_TOKEN")
    try {
        File("/path/to/data.csv").useLines { lines ->
            lines.forEach {
                try {
                    val line = it.split("|")
                    svix.eventType.create(EventTypeIn(line[1], line[0]))
                } catch (e: ApiException) {
                    println(e.message)
                    exitProcess(1)
                }
            }
        }
    } catch (e: IOException) {
        println(e.toString())
        exitProcess(1)
    }
}
```

**Review by svix-frank — APPROVED** [review]
I'll merge once the it hits maven

**Pull request merged**